specification/purview/data-plane/Azure.Analytics.Purview.MetadataPolicies/stable/2021-07-01/purviewMetadataPolicy.json

```
@@ -0,0 +1,535 @@
+          "$ref": "#/definitions/DNFCondition"
+        }
+      }
+    },
+    "CollectionReference": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "readOnly": true
+        },
+        "referenceName": {
+          "type": "string"
+        }
+      }
+    },
+    "MetadataPolicyProperties": {
+      "type": "object",
+      "properties": {
+        "description": {
+          "type": "string"
+        },
+        "decisionRules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DecisionRule"
+          }
+        },
+        "attributeRules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AttributeRule"
+          }
+        },
+        "collection": {
+          "$ref": "#/definitions/CollectionReference"
+        },
+        "parentCollectionName": {
+          "type": "string"
+        }
+      }
+    },
+    "MetadataPolicyViewModel": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string"
+        },
+        "version": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "properties": {
+          "$ref": "#/definitions/MetadataPolicyProperties"
+        }
+      }
+    },
+    "ErrorModel": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string"
+        },
+        "message": {
+          "type": "string"
+        },
+        "target": {
+          "type": "string"
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorModel"
+          }
+        }
+      }
+    },
+    "ErrorResponseModel": {
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorModel"
+        }
+      }
+    },
+    "MetadataPolicyViewModelListResults": {
+      "required": [
+        "values"
+      ],
+      "type": "object",
+      "properties": {
+        "values": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MetadataPolicyViewModel"
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      }
+    },
+    "RolePropertiesModel": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "type": "string"
+        },
+        "roleType": {
+          "type": "string"
+        },
+        "friendlyName": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        },
+        "cnfCondition": {
+          "$ref": "#/definitions/CNFCondition"
+        },
+        "dnfCondition": {
+          "$ref": "#/definitions/DNFCondition"
+        },
+        "version": {
+          "format": "int64",
+          "type": "integer"
+        }
+      }
+    },
+    "RoleViewModel": {
```

Simpler names are better.
```suggestion
    "Role": {
```